docker-compose.yml

```diff
@@ -1,13 +1,21 @@
 version: "3"
 services:
 
+#  nginx:
+#    image: nginx:1.15-alpine
+#    ports:
+#      - "80:80"
+#      - "443:443"
+#    volumes:
+#      - ./conf/nginx:/etc/nginx/conf.d
+
   nginx:
-    image: nginx:1.15-alpine
+    image: owasp/modsecurity
```

Comment:
why swap this?

Reply:
Ah shit, I didn't mean to include this (again) but I guess it's moot as we're not using the nginx container anyways. It was changed to use a `modsecurity` container that would sit "on top" of an nginx container. I think I committed this while testing it out on accident.